airbyte-integrations/bases/standard-test/standard_test/tests/test_core.py

```diff
@@ -69,9 +78,9 @@ def test_discover(self, connector_config, catalog, docker_runner: ConnectorRunne
             assert catalog_messages[0].catalog == catalog, "Catalog should match the one that was provided"
 
 
+@pytest.mark.timeout(300)
```

Comment:
I think not forcing timeout defeats the purpose of it. Another aspect is that we want to keep tests fast and scalable. They shouldn't take more than 20 minutes. If a connector can't guarantee this time either it is defective or it's test account configuration.

Reply:
I agree in principle, but pragmatically speaking I worry this may be difficult/full of friction to do in practice, so my suggestion is borne out of practicality rather than philosophy

Reply:
can we make it configurable? 300 second timeout sounds fine in this case

Reply:
as agreed with @sherifnada I will create a ticket for this, for now we will move on